### ISSUE TITLE
Horizons request limiter

### ISSUE THREAD

**Issue by mtbannister** [thespacedoctor/rockfinder#5]
Horizons now limits the number of lines that one can retrieve by asking it for a given number of distinct MJDs. (No, there wasn't a docs update...) 
This should ideally be raised to the user (asking for too many distinct MJDs), or alternatively implement a loop to retrieve all of the input MJDs.

**Comment by thespacedoctor** [comment]
Hi @mtbannister - do you need an urgent fix to this?

**Comment by mtbannister** [comment]
No, I'd engineered a work-around in my call to rockfinder, but wanted to let you know in case it affected your use of the function elsewhere. Would you like a PR?

**Comment by mtbannister** [comment]
(It's also only a problem for requesting >500 MJDs at once).
